Dharshika9/SPMBackend
# Task id #102 get students

how to test
1. use rest client and make a get request using 'http://localhost:8081/students' 
2. check whether registered student details are displaying in json format
